From 64817cd5563bb6f8de258844ce135efd68478469 Mon Sep 17 00:00:00 2001
From: Shaoyun Liu <Shaoyun.Liu@amd.com>
Date: Tue, 8 Aug 2017 10:40:24 -0400
Subject: [PATCH 1763/4131] drm/amdgpu: Remove unnecessary warning message

Same graphics BO can be imported mulitple times for interop

Change-Id: Iec45628f9c7df5edd7155f100f2756d68c515003
Signed-off-by: Shaoyun Liu <Shaoyun.Liu@amd.com>
---
 drivers/gpu/drm/amd/amdgpu/amdgpu_amdkfd_gpuvm.c | 3 ---
 1 file changed, 3 deletions(-)

diff --git a/drivers/gpu/drm/amd/amdgpu/amdgpu_amdkfd_gpuvm.c b/drivers/gpu/drm/amd/amdgpu/amdgpu_amdkfd_gpuvm.c
index bf01ad4..764585f 100644
--- a/drivers/gpu/drm/amd/amdgpu/amdgpu_amdkfd_gpuvm.c
+++ b/drivers/gpu/drm/amd/amdgpu/amdgpu_amdkfd_gpuvm.c
@@ -2320,9 +2320,6 @@ int amdgpu_amdkfd_gpuvm_restore_process_bos(void *info)
 		goto ttm_reserve_fail;
 	}
 
-	if (!list_empty(&duplicate_save))
-		pr_err("BUG: list of BOs to reserve has duplicates!\n");
-
 	amdgpu_sync_create(&sync_obj);
 	ctx.sync = &sync_obj;
 
-- 
2.7.4

